<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewGUI</class>
 <widget class="QMainWindow" name="ViewGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P2P Listen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0,5">
    <item>
     <widget class="QLabel" name="_labelYourIP">
      <property name="text">
       <string>Your LAN Address:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ipaddressLabel">
      <property name="text">
       <string>Loading Address...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="_labelSelectPort">
      <property name="text">
       <string>Send to:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="portLayout" stretch="8,1,4">
      <item>
       <widget class="QLineEdit" name="IPlineEdit">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="inputMask">
         <string>000.000.000.000;_</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_colon_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="portLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777205</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMask">
         <string>00000</string>
        </property>
        <property name="text">
         <string>45954</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSend">
      <property name="text">
       <string>Send File...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSend</sender>
   <signal>clicked()</signal>
   <receiver>ViewGUI</receiver>
   <slot>send_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>send_button_clicked()</slot>
 </slots>
</ui>
